packaging/examples/connect/kafka-connect-build.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+#  annotations:
+#  # use-connector-resources configures this KafkaConnect
+#  # to use KafkaConnector resources to avoid
+#  # needing to call the Connect REST API directly
+#    strimzi.io/use-connector-resources: "true"
```

Comment:
Should we enable connectors in this example? I guess it depends partly on the sorts of examples (e.g. in the docs) we might want to build around this YAML?

Reply:
I thought about that. It seemed better to follow what we have in the `kafka-connect.yaml` since the example might be also used without the connector resource. I can change it if everyone things it should be enabled by default here.

Reply:
WDYT? Should we keep it consistent with the `connect.yaml` example? Or should we enable it by default?